csharp/private/common.bzl

```diff
@@ -105,16 +47,19 @@ def fill_in_missing_frameworks(providers):
       deps: the original deps used to generate the providers.
     """
 
-    for gap_info in _find_framework_gaps(providers):
-        (base, compat_set_idx, start_idx, end_idx) = gap_info
+    # iterate through the compatability table since it's in preference order
```

Comment:
This is fine as long as `netcoreapp` doesn't start supporting `net` or vice-versa